code/game/gamemodes/cult/runes.dm

```diff
@@ -118,6 +120,8 @@ structure_check() searches for nearby cultist structures required for the invoca
 	//This proc determines if the rune can be invoked at the time. If there are multiple required cultists, it will find all nearby cultists.
 	var/list/invokers = list() //people eligible to invoke the rune
 	var/list/chanters = list() //people who will actually chant the rune when passed to invoke()
+	if(invisibility == 0)
```

Comment:
Pretty sure this does the opposite of what you want and either
A) does nothing because almost nothing ever becomes `0` invisibility
B) Means you can _only_ invoke when they're invisible